Setup.sh

```diff
@@ -19,7 +19,10 @@ BINARIES_DIR="$(pwd)/SpatialGDK/Binaries/ThirdParty/Improbable"
 SCHEMA_COPY_DIR="$(pwd)/../../../spatial/schema/unreal/gdk"
 SCHEMA_STD_COPY_DIR="$(pwd)/../../../spatial/build/dependencies/schema/standard_library"
 SPATIAL_DIR="$(pwd)/../../../spatial"
-
+DOMAIN_ENVIRONMENT_VAR=
```

Comment:
what is this doing? we are not using this variable anywhere in the script? Note that this variable will only be valid in the current script. as soon as Setup.sh finished execution, this will be lost

other quick comments: initialize the variable to an empty string ;) and it's usually safer to always have quotes around everything. 
can we change `$*` to `${1}` -> check only the first argument not all arguments